### PR TITLE
docs(json): add mode to docs-json docs

### PR DESCRIPTION
**Pull request by rwaskiewicz**
add an example of the output for modes associated with a stylesheet.

associated with https://github.com/ionic-team/stencil/pull/5718

STENCIL-1269

### CODE REVIEW

File: docs/documentation-generation/docs-json.md [hunk 1]
```diff
@@ -100,22 +100,48 @@ CSS file like the following:
...
+    "docs": "Background of the button when focused"
+  }
+]
+```
+
+If the style sheet is configured to be used with [a specific mode](../components/styling.md), the mode associated with
+the CSS property will be provided as well:
+
+```diff title="Example docs-json Output with Mode"
 "styles": [
```
Comment (christian-bromann) [inline docs/documentation-generation/docs-json.md:127]:
```suggestion
[
```

**Review by christian-bromann — COMMENTED** [review]
Worth removing this one too

File: docs/documentation-generation/docs-json.md (hunk 1 above)
Comment (rwaskiewicz) [reply docs/documentation-generation/docs-json.md]: Bah, I had that as a pending comment, but never submitted it. Thanks!

**Review by christian-bromann — APPROVED** [review]
LGTM 👍

**Review by alicewriteswrongs — APPROVED** [review]

**Comment by rwaskiewicz** [comment]
Thanks all! Since this is new functionality, no backport of the docs needed.

**Pull request merged**